public/app/features/dashboard/components/PanelEditor/PanelEditor.tsx

```diff
@@ -62,15 +66,28 @@ type Props = OwnProps & ConnectedProps & DispatchProps;
 
 export class PanelEditorUnconnected extends PureComponent<Props> {
   querySubscription: Unsubscribable;
+  rafToken = createRef<number>();
 
   componentDidMount() {
     this.props.initPanelEditor(this.props.sourcePanel, this.props.dashboard);
+
+    window.addEventListener('resize', debounce(this.updateSplitPaneSize, 100));
```

Comment:
I think the debounce is not necessary here. We should use **either** debounce or rAF